src/Sulu/Bundle/WebsiteBundle/Cache/Events.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\WebsiteBundle\Cache;
```

Comment:
Based on the Symfony Best Practice a Bundle should contain all Events in a Events Class in the Root of its Bundle.

Reply:
I moved the Events class.